app/Http/Controllers/EmployeeController.php

```diff
@@ -2,69 +2,108 @@
+                              ->orWhere('dob' ,'like' , "%".$search."%")
+                              ->orWhere('phone' ,'like' , "%".$search."%")
+                              ->orWhereHas('bankAccount',
+                              function ($q) use ($search) {
+                                  $q->where('account_no', 'like', "%".$search."%")->select('branch'); })
+                              ->orWhereHas('company', function ($q) use ($search) {
+                                  $q->where('name', 'like', "%".$search."%"); })
+                              ->orWhereHas('company', function ($q) use ($search) {
+                                  $q->where('branch', 'like', "%".$search."%"); });
+
+          $filteredEmployees = $search ? $employees->count() : $totalEmployees;
+          $employees = $employees->skip($start)
+                                  ->take($length)
+                                  ->get();
 
-        //SERVER SIDE RENDERING HANDLE FOR data table
-        $search = $request->query('search')['value'];
-        $draw = $request->query('draw', 1);
-        $start = $request->query('start', 0);
-        $length = $request->query('length', 10);
-        $totalEmployees =   Employee::count();
-
-        $employees = Employee::with('bankAccount' , 'company')->where('name' , 'like' , "%".$search."%")
-                            ->orWhere('position' ,'like' , "%".$search."%")
-                            ->orWhere('email', 'like' , "%".$search."%")
-                            ->orWhere('address' ,'like' , "%".$search."%")
-                            ->orWhere('dob' ,'like' , "%".$search."%")
-                            ->orWhere('phone' ,'like' , "%".$search."%")
-                            ->orWhereHas('bankAccount',
-                            function ($q) use ($search) {
-                                $q->where('account_no', 'like', "%".$search."%")->select('branch'); })
-                            ->orWhereHas('company', function ($q) use ($search) {
-                                $q->where('name', 'like', "%".$search."%"); });
-
-        $filteredEmployees = $search ? $employees->count() : $totalEmployees;
-        $employees = $employees->skip($start)
-                                ->take($length)
-                                ->get();
-
-        $response = [
-            'draw' => intval($draw),
-            'recordsTotal' => intval($totalEmployees),
-            'recordsFiltered' => $filteredEmployees,
-            'data' => $employees
-        ];
-
-        return Response::json($response);
+          $response = [
+              'draw' => intval($draw),
+              'recordsTotal' => intval($totalEmployees),
+              'recordsFiltered' => $filteredEmployees,
+              'data' => $employees
+          ];
+          return Response::json($response);
     }
 
+
     public function store(Request $request){
+        $errors = [];
```

Comment:
add one block of try catch and return the error from catch block. try catch will be one for entire method. remove the if conditions